src/styles/themes/dark.ts

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable no-unused-vars */
+import colors from '../colors';
+import SCREENS from '../../SCREENS';
+import {HexColor, ThemeColors} from './ThemeColors';
+
+const darkTheme: ThemeColors = {
+    // Figma keys
+    appBG: colors.darkAppBackground,
+    splashBG: colors.green400,
+    highlightBG: colors.darkHighlightBackground,
+    border: colors.darkBorders,
+    borderLighter: colors.darkDefaultButton,
+    borderFocus: colors.green400,
+    icon: colors.darkIcons,
+    iconMenu: colors.green400,
+    iconHovered: colors.darkPrimaryText,
+    iconSuccessFill: colors.green400,
+    iconReversed: colors.darkAppBackground,
+    iconColorfulBackground: `${colors.ivory as HexColor}cc`,
```

Comment:
Isn't better to define a separate color for this?

Reply:
You're right, but it's just string now anyway 👍